hs_core/forms.py

```diff
@@ -190,22 +225,28 @@ def __init__(self, allow_edit=True, res_short_id=None, element_id=None, *args, *
 
     @property
     def form_id(self):
+        """Render proper form id by prepending 'id_creator_'."""
         form_id = 'id_creator_%s' % self.number
         return form_id
 
     @property
     def form_id_button(self):
+        """Render proper form id with quotes around it."""
         form_id = 'id_creator_%s' % self.number
         return "'" + form_id + "'"
 
     class Meta:
+        """Describe meta properties of PartyForm."""
+
         model = Creator
         fields = PartyForm.Meta.fields
         fields.append("order")
         labels = PartyForm.Meta.labels
 
 
 class PartyValidationForm(forms.Form):
+    """Render validated form for Party models."""
```

Comment:
This form (and other validation forms) is used only for validating user posted data for creating/updating party models. This form is not used for rendering.